### DISCUSSION TITLE
cara mengubah dimensi input

### DISCUSSION THREAD

**Discussion by Gustana** [teamalgoritma/community#735] [Q&A] [answered]
![image](https://github.com/teamalgoritma/community/assets/23212456/219a82f8-4686-44cf-98e8-6d6f7535baa9)

**Comment by divakartika** [comment, marked as answer]
Dear @Gustana ,

Kemungkinan terdapat kesalahan dalam penulisan parameter `input_shape` atau pembentukan dimensi prediktor yang kurang tepat. Berikut beberapa referensi yang dapat kami berikan.
- [How to properly set the input_shape of LSTM layers?](https://stackoverflow.com/questions/61153831/how-to-properly-set-the-input-shape-of-lstm-layers)
- [Understanding input_shape parameter in LSTM with Keras](https://stats.stackexchange.com/questions/274478/understanding-input-shape-parameter-in-lstm-with-keras)
- [Keras LSTM Diagram](https://github.com/MohammadFneish7/Keras_LSTM_Diagram)

Semoga membantu dan menjawab pertanyaan Anda :)

**Comment by Gustana** [comment]
berikut contoh lstm model dari [algotech](https://algotech.netlify.app/blog/text-lstm/)
![image](https://github.com/teamalgoritma/community/assets/23212456/9de3057a-b578-43ea-b624-8995a217c8d0)

berikut pembentukan dan summary model saya,
![image](https://github.com/teamalgoritma/community/assets/23212456/b7de4cc6-1f4e-4501-bd0d-544afc0810e4)
![image](https://github.com/teamalgoritma/community/assets/23212456/c7de4762-8940-4860-9377-73a7de1ec4d8)

apakah ada yang salah dari input shapenya?

**Reply by IrfanChairurrachman** [reply]
Dear @Gustana ,

Apakah boleh tau tujuan dilakukannya penyatuan `train_text_seq` menggunakna `sapply` dan `paste` di bagian kode ini?

![image](https://github.com/teamalgoritma/community/assets/56690035/d6e01b91-937e-4722-8808-4b1fa9335edc)

Dan jika boleh tau apakah telah melakukan training menggunakan data train tersebut Mas?

**Reply by kevbow** [reply]
Halo @Gustana, untuk menjaga agar forum diskusi tetap berjalan kondusif, kami izin melakukan lock/close diskusi ini. Semoga dapat dipahami.
